### ISSUE TITLE
After installing a new binary, still uses old live deploy snapshot.

### ISSUE THREAD

**Issue by valterkraemer** [ionic-team/cordova-plugin-ionic#179]
**cordova plugin ls output:**
```
cordova-clipboard 1.2.1 "Clipboard"
cordova-plugin-advanced-http 2.0.1 "Advanced HTTP plugin"
cordova-plugin-app-version 0.1.9 "AppVersion"
cordova-plugin-appavailability 0.4.2 "AppAvailability"
cordova-plugin-camera 4.0.3 "Camera"
cordova-plugin-contacts 3.0.1 "Contacts"
cordova-plugin-device 2.0.2 "Device"
cordova-plugin-file 6.0.1 "File"
cordova-plugin-file-transfer 1.7.1 "File Transfer"
cordova-plugin-inappbrowser 3.0.0 "InAppBrowser"
cordova-plugin-ionic 5.2.6 "cordova-plugin-ionic"
cordova-plugin-ionic-keyboard 2.1.3 "cordova-plugin-ionic-keyboard"
cordova-plugin-ionic-webview 2.2.0 "cordova-plugin-ionic-webview"
cordova-plugin-splashscreen 5.0.2 "Splashscreen"
cordova-plugin-statusbar 2.4.2 "StatusBar"
cordova-plugin-stripe 1.5.3 "cordova-plugin-stripe"
cordova-plugin-whitelist 1.3.3 "Whitelist"
cordova-plugin-x-socialsharing 5.4.1 "SocialSharing"
cordova-sqlite-storage 2.5.0 "Cordova sqlite storage plugin"
es6-promise-plugin 4.2.2 "Promise"
ionic-plugin-deeplinks 1.0.17 "Ionic Deeplink Plugin"
onesignal-cordova-plugin 2.4.3 "OneSignal Push Notifications"
```

**ionic info output:**
```
Ionic:

   ionic (Ionic CLI)  : 4.2.1
   Ionic Framework    : ionic-angular 3.9.2
   @ionic/app-scripts : 3.2.0

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : android 7.1.1, ios 4.5.5
   Cordova Plugins       : cordova-plugin-ionic 5.2.6, cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 2.2.0, (and 18 other plugins)

System:

   Android SDK Tools : 26.1.1 (/Users/xyz/Library/Android/sdk)
   ios-deploy        : 2.0.0
   NodeJS            : v8.12.0 (/Users/xyz/.nvm/versions/node/v8.12.0/bin/node)
   npm               : 6.4.1
   OS                : macOS High Sierra
   Xcode             : Xcode 9.4.1 Build version 9F2000
```

**Describe the bug**
After installing a new binary, still uses old live deploy snapshot.

**To Reproduce**
1. Install binary with v2.0.6
2. Allow Live update to update it to v2.0.8
3. Update binary to v2.1.2
4. Still have v2.0.8

**Expected behavior**
In step 4 have v2.1.2

**Other info**
* In Ionic Pro dashboard, maximum version is set to 2.0.7.
* Versions are set in `package.json`, `config.xml` and `Pro.init(`.
* UPDATE_METHOD=none.
* `cordova plugin ls` and `ionic info` outputs are for v2.1.2. Differs a bit in v2.0.8.

This is what I see in console when I start v2.1.2:

<img width="803" alt="screen shot 2018-11-01 at 11 06 15" src="https://user-images.githubusercontent.com/5527296/47843020-0b24ed80-ddc7-11e8-9e25-3732b8967aae.png">

**Comment by jcesarmobile** [comment]
As you are using none as update method, can you share the code you are using to check/apply the updates?

**Comment by valterkraemer** [comment]
```
await this.platform.ready();
const version = await this.appVersion.getVersionNumber();
const data = await this.http.get<any>(`${API_URL}api/version`)
  .toPromise();

if (this.isNewer(data.version, version)) {
  // Code that shows dialog to download new binary
  return;
}

await Pro.deploy.configure({
  channel: DEPLOY_CHANNEL
});

const config = await Pro.deploy.getConfiguration();
console.log('config', JSON.stringify(config));

const update = await Pro.deploy.checkForUpdate();
console.log('update', JSON.stringify(update));

if (update.available) {
  let loadingAlert = this.loadingCtrl.create({
    spinner: 'bubbles',
    content: 'Updating app.'
  });
  loadingAlert.present();

  await Pro.deploy.downloadUpdate((progress) => {
    loadingAlert.data.content = `Updating app. ${progress}% completed.`;
  });

  await Pro.deploy.extractUpdate();
  await Pro.deploy.reloadApp();
  loadingAlert.dismiss();
}
```

**Comment by nphyatt** [comment]
@valterkraemer found the issue and just released `5.2.7` which should fix this issue.
